tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -202,13 +204,18 @@ internal void UpdateAddedProbeInstrumentations(IReadOnlyList<ProbeDefinition> ad
                             var nativeDefinition = new NativeMethodProbeDefinition(probe.Id, probe.Where.TypeName, probe.Where.MethodName, probe.Where.Signature?.Split(separator: ','));
                             methodProbes.Add(nativeDefinition);
                             break;
+                        case ProbeLocationType.MethodSpan:
+                            var spanProbe = probe as SpanProbe;
+                            var spanDefinition = new NativeSpanProbeDefinition(probe.Id, probe.Where.TypeName, probe.Where.MethodName, probe.Where.Signature?.Split(separator: ','));
```

Comment:
Do we need to trim whitespace here (and in line 204)?

Reply:
That's a good question. I'll test it out